ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -204,8 +300,12 @@ spec:
           ports:
           - name: http
             containerPort: 80
+            hostPort: 38080
+            protocol: TCP
           - name: https
             containerPort: 443
+            hostPort: 38
```

Comment:
Also needs to be updated to 38443 since that was a typo that was corrected under the config file.

Reply:
Made two updates - line 303 and 307. Thank you!